MODULE.bazel

```diff
@@ -6,7 +6,7 @@ module(
     compatibility_level = 1,
 )
 
-bazel_dep(name = "aspect_bazel_lib", version = "1.31.0")
+bazel_dep(name = "aspect_bazel_lib", version = "1.29.2")
```

Comment:
Can this be updated to `1.30.2`?

Reply:
Oh this is a revert PR. Nevermind.

Reply:
it shouldn't be. We need to take a different policy with updating our MODULE.bazel deps - we ought to be giving the lower bound here so we don't churn users. I'll post something on slack.